.github/actions/setup/action.yml

```diff
@@ -4,6 +4,10 @@ description: Setup the environment for the other workflows
 runs:
   using: 'composite'
   steps:
+    - name: Mark workspace as safe
```

Comment:
what caused this to be needed? I spend time yesterday trying to sort out the permissions in the container so that this and other workarounds weren't needed

Reply:
The job failed immediately without this.